tool/clean/clean_file_system/clean_file_system.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Instead of creating a separate file, it would be good to share the common resources or provide general resources that can be shared between cleaning functionality (e.g daysToKeep)

Reply:
The functionality is not common. I did create a util file for common fucntuonality.